creator.html

```diff
@@ -10,15 +10,10 @@
                 <div class="mask rgba-black-light d-flex justify-content-center align-items-center">
                     <div class="text-center white-text mx-5 wow fadeIn">
                         <h1 class="mb-4">
-                            <strong>Convert your ideas into apps with Kodular!</strong>
```

Comment:
Are you sure we want to capitalize each letter?

Reply:
Yeah, I think it's looking much better now.
![screenshot-www kodular io-2019 05 02-09-39-15](https://user-images.githubusercontent.com/40118018/57056178-5f4d5380-6cbe-11e9-9c7f-39f03bd1a3a3.png)

Reply:
I think it's better as it was before; it has more than 5 words